### DISCUSSION TITLE
來了! AI工程師誕生了

### DISCUSSION THREAD

**Discussion by tzurou** [1101-GS4719-C/Python-Community-GS4719-C#1936] [General]
全球的AI旋風正在如火如荼的進行中，我最近也看到一則新聞，內容是關於世界上第一位AI工程師的誕生，他不僅會寫程式、除錯，還能建立網站和編寫軟體，並且文章內容強調AI工程師並不是為了取代人類工程師，而是要與人類合作讓我們的生活更便利，然而我認為AI工程師的誕生一定會不可避免的取代許多現有工程師的工作，儘管近幾年掀起跨領域學習的風潮，但我相信還有許多人類工程師仍做著AI工程師也能完成的工作，而他們終將被AI工程師淘汰，我自己是文組所以對理工領域不太了解，單純是就我所知闡述我的想法，不知道大家怎麼看這件事?
附上文章連結：https://news.cnyes.com/news/id/5487575

**Comment by 110101501** [comment]
其實不單只有工程師領域，其他領域與AI的應用層面我相信在未來也會愈來愈廣。我所閱讀過的文章不少也是強調說AI不是為了取代人類，而是輔助人類的生活。舉例而言，AI輔助檢測阿茲海默症患者失智前期徵兆、抑或是神經叢手術的應用，就是AI在醫療領域與人類合作使生活更好。

**Comment by jylin34** [comment]
我是覺得ai不會完全取代軟體工程師的工作，他扮演的角色更像是"加速器"，軟體工程領域會隨著ai的發展 越來越抽象化，現在工程師大部分在寫程式，以後變成再跟ai下prompt。未來公司可能只需要聘一個人+ai就能達成以往三個人的工作量，ai會變成人人都要會的技能，但同時也會大幅提升科技公司開發各種產品的效率。
資工系的路過，難道畢業前就要被devin取代了嗎...

**Comment by 20050201Shawn** [comment]
我是非資工科系的學生，我覺得ai確實無法避免的會取代某些人正在做的工作，但不代表會完全取代人類。目前的ai比較偏向能取代重複性高、勞力性質的工作，雖然現在ai好像進步太快了，不只會編程式也會生成藝術，但我認為創新是最關鍵的能力，這些現有的ai模型也是依照已存在的資料去學習、模仿，雖然有時候覺得我們打的文章只是各個字的排列組合而已，不同曲子也是各音符排列組合不同的結果，就算沒有ai，我們仍能察覺人類所創造的一切都是排列組合(?)，有一種固定的公式和習慣，像是議論文的結構就是敘論、論證、論據。所以創新的能力能使我們脫離現有的框架，給予新的活力和靈感。最後舉個例子，我們不會說汽車取代我們的雙腳，而是輔助我們能去更遠的地方，然而汽車也取代了以往的人力車。

**Comment by SusuTima** [comment]
我覺得就如同文章所說，AI工程師不是用來取代人類工程師的，而是為了與人類工程師共同合作。在未來也很樂見AI在各個領域的應用，有了AI的幫助，相信許多先前認為繁雜的工作，都可以變得得心應手。

**Comment by willychiuchiu** [comment]
雖然說AI工程師的出現並不會完全取代軟體工程師，但我也非常認同版主所說的，其誕生一定不可避免的會取代很多人類工程師，在同樣的工作量下，所需要的軟體工程師會大幅下降，況且Devin的能力已經有著平均之上的水平，所以說不會取代的大概都是在安慰自己而已，只能說人類要更努力進步，學著與AI合作，提升自己的價值，不讓自己被AI淘汰！

**Comment by benbebobo149** [comment]
提到這篇新聞，我就想到這家公司的執行長是一個非常厲害的人。 在中學時期便參加了國際奧林匹亞 計算機組拿到三次金牌，當我以為他會穩穩去讀資工的時候，沒想到他在大學的主修科目竟是經濟。 這真的讓我感到驚訝，因為通常人們會認為他會在自己的專業領域發展。也許是他擁有廣闊的視野和跨領域的思維，讓他能夠發現商機並有足夠的企圖心，去實現一個前人從未覺得是可行的項目。這個故事讓我深深地感到啟發，謝謝同學的分享，讓我有機會了解到這樣的故事。

**Comment by Hahahahbla** [comment]
你提出的觀點是很值得討論的。確實，AI的發展可能會影響一些傳統的工作，但同時也會創造新的機會和需求。AI工程師的出現可能會改變工程領域的需求和技能，但這也可以看作是一種轉型和進步。重要的是要提供給現有工程師學習和轉型的機會，以應對未來的挑戰。AI工程師和人類工程師可以共同合作，彼此補充，實現更大的價值。

**Comment by OG-Matcha** [comment]
我想就算之後真的有 AI 工程師也不會完全淘汰掉原本的軟體工程師，如果都交給 AI 那誰來撰寫這些 AI 產品呢。他起到的應是加快我們工作生產效率的作用，會影響到的是那些複雜性低、重複性高的工作內容，還有會應用 AI 與不會應用 AI 的人就會有很明顯的鑑別度，一方面降低了進入這個領域的門檻，另一方面也拉高了要達到「出類拔萃」程度的天花板。另外不管今天 AI 發展到甚麼程度，總歸是需要由「人」去進行「關鍵決策」，這才是我們的價值。

**Comment by NcuJerryKao** [comment]
這篇文章探討了AI技術的進步及其對人類工程師的影響。以下是幾個建設性建議，幫助提升文章的結構、內容和清晰度：
1. 引言部分的強化：
- 引言可以更具吸引力和信息性，簡要介紹AI技術的發展背景和文章的主題。例如，可以提到AI在各行各業中的應用及其對職場的影響。
- 例如：「全球AI技術的迅猛發展正在改變各行各業的面貌。最近，一則關於世界上第一位AI工程師誕生的新聞引發了廣泛關注。這位AI工程師不僅能寫程式、除錯，還能建立網站和編寫軟體。」
2. 段落結構：
- 將文章分成更易於理解的段落，每個段落集中討論一個具體的觀點。例如：
1. 引言：介紹AI技術的發展背景和文章主題。
2. AI工程師的能力：詳細說明AI工程師具備的技能和特點。
3. 人類工程師的擔憂：探討AI工程師對人類工程師職位的影響。
4. 合作與取代：分析AI工程師與人類工程師之間的合作與競爭。
5. 結論和建議：總結文章並提出個人觀點和建議。
3. 過渡和連接詞：
- 使用過渡句和連接詞來連接不同段落，使文章流暢。例如，「接下來，我們來詳細了解這位AI工程師的能力和特點。」或「然而，AI工程師的誕生也引發了人類工程師對職位安全的擔憂。」
4. 詳細說明和分析：
- 在描述AI工程師的能力時，可以更詳細地解釋其具體技能和應用場景。例如，列出AI工程師能完成的具體任務，如程式編寫、網站建立和軟體開發。
- 例如：「這位AI工程師不僅能熟練地編寫程式和除錯，還能獨立建立網站和開發各類軟體，展示了強大的技術能力。」
5. 語言簡化和澄清：
- 某些段落中的語言較為簡單，可以考慮稍微豐富和澄清。例如，「AI工程師的誕生一定會不可避免的取代許多現有工程師的工作」可以改為「儘管AI工程師被設計來與人類合作，但其強大的能力可能會取代許多人類工程師的工作。」
6. 引用和支持：
- 若能引用一些具體的數據、報告或專家觀點來支持論點，將大大增強文章的權威性。例如，可以引用科技報告或專家觀點來支持「AI工程師對職場的影響」的討論。
7. 個人經驗和反思：
- 若有相關的個人經驗或反思，可以分享在文章中，這樣可以讓讀者更容易產生共鳴。例如，描述自己作為文組學生對AI技術的觀察和感受。
8. 結論和資源提供：
- 在結論部分，再次強調文章的主要觀點，並提供進一步學習的資源。這有助於讀者記住文章的核心思想，並了解如何繼續學習。
- 例如：「總結來說，AI工程師的誕生既帶來了技術進步的機遇，也引發了職場變革的挑戰。我們需要積極應對這一變化，提升自身技能，以在未來的職場中保持競爭力。」
9. 讀者互動：
- 在文章結尾，可以提出一些問題或邀請讀者分享他們的觀點和經驗，以增加互動性和參與感。
- 例如，「你對AI工程師的誕生有什麼看法？你認為AI技術會對你的職業產生什麼影響？歡迎在評論區分享你的意見。」
通過這些建議，文章將變得更加結構化、清晰且具說服力，讀者也會更容易理解和應用其中的信息。

**Comment by titan92601** [comment]
我認為AI工程師的興起確實可能改變工作模式，但人類工程師仍有獨特價值。創造力、情感等特質是AI無法取代的。因此，雖然部分工作可能被自動化，但人類工程師的角色仍然重要。這也提醒我們，持續學習和進步對於在快速變化的科技世界中保持競爭力至關重要。

**Comment by alexender0104** [comment]
我認為AI工程師的出現代表著科技領域的一個重大進步。這些工程師專注於開發、設計和維護人工智慧和機器學習系統，並應用這些技術解決各種現實世界的問題。他們的工作為推動科技創新和解決現實世界的難題做出了重要貢獻。

**Comment by hbh10601** [comment]
對我來說，AI工程師還是需要人類工程師去維護的，除非哪天可以透過AI去維護AI，可能才會真的完全被取代，還有可能被AI統治(?

**Comment by sandyho0307** [comment]
我覺得像 Devin 這樣的 AI 工具可以幫助工程師減少繁瑣的重複性工作，讓他們有更多的時間專注於創新和高階的技術開發。此外，這也反映出市場對於跨領域、多技能人才的需求越來越高。

**Comment by yubo0oo** [comment]
我覺得，雖然這個觀點很理想，但現實中，AI工程師的誕生必然會取代許多現有工程師的工作。儘管近幾年掀起了跨領域學習的風潮，但我相信還有很多人類工程師依然在做著AI工程師也能完成的工作，這些工作終將被AI取代。

我自己是文組，所以對理工領域不太了解，這只是就我所知闡述我的想法。我認為這是一個不可避免的趨勢，但也希望AI的發展能真正實現人機合作，而不是單純的取代。大家對這件事怎麼看呢？歡迎分享你的看法！****

**Comment by JustAHedgehog** [comment]
其實類似議題在其他篇文章也有提及，這邊提供傳送門讓有興趣的人可以多加了解，看看大家的想法是否雷同。
除了我的想法外，這邊也分享Pypal提供的想法：
工程師的角色將會逐漸轉變，尤其是隨著科技的發展，我們需要不斷學習新技能來跟上時代。 ChatGPT 確實可以幫助主管快速撰寫程式碼，但這也代表著工程師需要更多的創造力、解決問題的能力和對新技術的掌握。留下來的工程師將是具備全方位能力的人才，才能在未來的競爭中脫穎而出。不要害怕改變，持續學習、成長，你一定能成為那個能夠應對未來挑戰的工程師！
[傳送門]https://github.com/NCU-GS4719-Python/Python-Community/discussions/908)
